app/src/main/java/org/p2p/wallet/history/HistoryModule.kt

```diff
@@ -80,12 +81,7 @@ object HistoryModule : InjectionModule {
 
     private fun Module.dataLayer() {
         factory { TransactionDetailsEntityMapper(get()) }
-        single {
-            TransactionDetailsDatabaseRepository(
-                daoDelegate = get(),
-                mapper = get()
-            )
-        } bind TransactionDetailsLocalRepository::class
+        singleOf(::TransactionDetailsDatabaseRepository) bind TransactionDetailsLocalRepository::class
```

Comment:
```suggestion
        singleOf<TransactionDetailsLocalRepository>(::TransactionDetailsDatabaseRepository)
```

Reply:
Imho the keyword `bind` looks better than the way you suggest. 
Because it may be confusing which type are you returning
@gslevinkov

Reply:
For me both cases are ok but I like Gleb's points. Second way is shorter and looks more idiomatic

Reply:
I prefer binds 
but can live with second way :)

Reply:
i prefer <Type> for me it's more clear, cause we already use this arrayListOf<Type> and etc

Reply:
![1](https://user-images.githubusercontent.com/19359742/174567568-590aa0ab-f417-4e8c-b405-6a05d0009efb.png)
looks like it's not supported guyz! 
Merging this, and will decide what to do in the future